WordPress/src/main/java/org/wordpress/android/models/ReaderTagList.java

```diff
@@ -60,9 +63,9 @@ public ReaderTagList getDeletions(ReaderTagList tagList) {
             return deletions;
         }
 
-        for (ReaderTag thisTag: this) {
-            if (!tagList.hasSameTag(thisTag)) {
-                deletions.add(thisTag);
+        for (ReaderTag tag: this) {
```

Comment:
The comment above is probably wrong. If we need to return a list of tags that are in this list but not in the passed list
`indexOfTag` should be called on `tagList`.